src/Commands/BaseCommand.php

```diff
@@ -94,6 +94,14 @@ public function landoSetup()
                 ->to($site_name)
                 ->run();
 
+            $siteInfo = $this->getSite($site_name)->serialize();
```

Comment:
The `$site_name` is basically the repo name/folder name. Is this approach correct or we will need to find out the correct `$site_name` and then the respective `uuid`.

Reply:
I'm not sure this will always be the case - I think the site name can be passed as an argument to the project create command. But I think the logic here still works.

Reply:
Or I'm understanding it wrong. We will need uuid of any existing site which in pantheon admin dashboard?

Reply:
For this use case, we're creating a brand new site. So we won't be able to use an existing site name.

Reply:
For creating a brand new site we will run the `terminus build:project:create` which is not part of this plugin's command. 
So I think we should make changes to [terminus-build-tools-plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin) where we can may be create some endearment value to have the UUID or the site name stored.
Not too sure about that though.

Reply:
Given this functionality will be only for Decoupled template based projects. I was thinking to pass the respective site name with the command `terminus build:lando:setup` and then we can get the uuid for it.

Reply:
@abhisekmazumdar Not positive, but I suspect that we don't need to make changes to the project create command. This plugin introduces a `terminus lando:setup` command that runs before the site is created. The problem we're facing now is that we won't know the site ID at that point. I'd approach this by trying to adjust when that script actually runs.

This script in decoupled-drupal https://github.com/pantheon-systems/decoupled-drupal/blob/master/scripts/composer/project-tasks is what actually executes the command (you'll see a comment there explaining that this approach is being used so we don't have to modify the project create command). The composer scripts https://github.com/pantheon-systems/decoupled-drupal/blob/master/composer.json#L52-L71 also influence what is run when. If we can ensure that lando command runs after the site is created, we should be able to determine the site id.